### ISSUE TITLE
Filenames not extracted properly

### ISSUE THREAD

**Issue by himynamesdave** [signalscorps/file2stix#26]
filenames and directories included in a single object, e.g.

  "pattern": "[ file:name = 'tests/file_inputs/observable_tests/asn.txt' ]",

This should be a filename (asn.txt) and directory (tests/file_inputs/observable_tests/)

tests/file_inputs/observable_tests/asn.txt'

to recreate 

```
obstracts-cli --input-file tests/file_inputs/observable_tests/asn.txt
```

```
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--241b6505-8250-4a9e-8f61-c4d2881141eb",
            "created": "2022-08-23T15:34:14.184953Z",
            "modified": "2022-08-23T15:34:14.184953Z",
            "name": "File name: tests/file_inputs/observable_tests/asn.txt",
            "indicator_types": [
                "malicious-activity"
            ],
            "pattern": "[ file:name = 'tests/file_inputs/observable_tests/asn.txt' ]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-08-23T15:34:14.184953Z"
        },
```

**Comment by himynamesdave** [comment]
Yes, sorry I was not more clear @shanthanu9 

See above a Indicator (file) extracted

```
"pattern": "[ file:name = 'tests/file_inputs/observable_tests/asn.txt' ]"
```

However, this should be extracted as two Indicators, one file, one directory.

The file should be asn.txt

```
"pattern": "[ file:name = 'asn.txt' ]"
```

And the path should be a directory

```
"pattern": "[ directory:path = 'tests/file_inputs/observable_tests/' ]"
```

**Comment by himynamesdave** [comment]
@shanthanu9 try

stixify --input-file tests/file_inputs/observable_tests/directory.txt

See in the bundle below filenames still being included as directories

"pattern": "[ file:name = '/System/Library/LaunchDaemons/test.bat' ]"

```
{
    "type": "bundle",
    "id": "bundle--953b5ed2-1b12-4e5d-acd9-9fda9211a5ee",
    "objects": [
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--1ce7a563-c3bc-4ae1-af88-ac7b816fefd0",
            "created": "2022-08-25T09:26:04.668095Z",
            "modified": "2022-08-25T09:26:04.668095Z",
            "name": "File name: /System/Library/LaunchDaemons/test.bat",
            "indicator_types": [
                "unknown"
            ],
            "pattern": "[ file:name = '/System/Library/LaunchDaemons/test.bat' ]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-08-25T09:26:04.668095Z"
        },
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--a8919108-2e57-4b44-8f67-5d081e35bd9f",
            "created": "2022-08-25T09:26:10.882247Z",
            "modified": "2022-08-25T09:26:10.882247Z",
            "name": "File name: C:\\Windows\\System32\\test.bat",
            "indicator_types": [
                "unknown"
            ],
            "pattern": "[ file:name = 'C:\\\\Windows\\\\System32\\\\test.bat' ]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-08-25T09:26:10.882247Z"
        },
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--f9773c8b-23e3-4b7e-9b51-8e5d1b1886f2",
            "created": "2022-08-25T09:26:12.883585Z",
            "modified": "2022-08-25T09:26:12.883585Z",
            "name": "File name: tests/file_inputs/observable_tests/directory.txt",
            "indicator_types": [
                "unknown"
            ],
            "pattern": "[ file:name = 'tests/file_inputs/observable_tests/directory.txt' ]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-08-25T09:26:12.883585Z"
        },
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--1ea38146-d049-4dbd-bd57-4b2bac6b1783",
            "created": "2022-08-25T09:26:15.235819Z",
            "modified": "2022-08-25T09:26:15.235819Z",
            "name": "Directory: /System/Library/LaunchDaemons",
            "indicator_types": [
                "unknown"
            ],
            "pattern": "[ directory:path = '/System/Library/LaunchDaemons' ]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-08-25T09:26:15.235819Z"
        },
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--d7c1534d-02fe-4278-91ba-b9c6cbb2e3bd",
            "created": "2022-08-25T09:26:17.819945Z",
            "modified": "2022-08-25T09:26:17.819945Z",
            "name": "Directory: /System/Library/LaunchDaemons/test.bat",
            "indicator_types": [
                "unknown"
            ],
            "pattern": "[ directory:path = '/System/Library/LaunchDaemons/test.bat' ]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-08-25T09:26:17.819945Z"
        },
        {
            "type": "tool",
            "id": "tool--0c8465c0-d0b4-4670-992e-4eee8d7ff952",
            "created_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5",
            "created": "2017-05-31T21:33:06.824Z",
            "modified": "2022-04-20T20:02:41.550Z",
            "name": "at",
            "description": "[at](https://attack.mitre.org/software/S0110) is used to schedule tasks on a system to run at a specified date or time.(Citation: TechNet At)(Citation: Linux at)",
            "labels": [
                "tool"
            ],
            "external_references": [
                {
                    "source_name": "mitre-attack",
                    "url": "https://attack.mitre.org/software/S0110",
                    "external_id": "S0110"
                },
                {
                    "source_name": "Linux at",
                    "description": "IEEE/The Open Group. (2017). at(1p) \u2014 Linux manual page. Retrieved February 25, 2022.",
                    "url": "https://man7.org/linux/man-pages/man1/at.1p.html"
                },
                {
                    "source_name": "TechNet At",
                    "description": "Microsoft. (n.d.). At. Retrieved April 28, 2016.",
                    "url": "https://technet.microsoft.com/en-us/library/bb490866.aspx"
                }
            ],
            "object_marking_refs": [
                "marking-definition--fa42a846-8d90-4e51-bc29-71d5b4802168"
            ],
            "x_mitre_aliases": [
                "at",
                "at.exe"
            ],
            "x_mitre_attack_spec_version": "2.1.0",
            "x_mitre_deprecated": false,
            "x_mitre_domains": [
                "enterprise-attack"
            ],
            "x_mitre_modified_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5",
            "x_mitre_platforms": [
                "Linux",
                "Windows",
                "macOS"
            ],
            "x_mitre_version": "1.2"
        },
        {
            "type": "report",
            "spec_version": "2.1",
            "id": "report--2f71228b-f317-4029-ad42-0e98f599ad1d",
            "created": "2022-08-25T09:26:33.920216Z",
            "modified": "2022-08-25T09:26:33.920216Z",
            "name": "File converted: directory.txt",
            "report_types": [
                "threat_report"
            ],
            "published": "2022-08-25T10:26:33.920183Z",
            "object_refs": [
                "indicator--1ce7a563-c3bc-4ae1-af88-ac7b816fefd0",
                "indicator--a8919108-2e57-4b44-8f67-5d081e35bd9f",
                "indicator--f9773c8b-23e3-4b7e-9b51-8e5d1b1886f2",
                "indicator--1ea38146-d049-4dbd-bd57-4b2bac6b1783",
                "indicator--d7c1534d-02fe-4278-91ba-b9c6cbb2e3bd",
                "tool--0c8465c0-d0b4-4670-992e-4eee8d7ff952"
            ]
        }
    ]
}
```
